### ISSUE TITLE
Incorrect loss for AutoEncoder

### ISSUE THREAD

**Issue by ncoop57** [timsainb/tensorflow2-generative-models#9]
In computing the testing loss for the autoencoder, you reuse the training set

**Comment by timsainb** [comment]
Thanks for pointing this out, it looks like I did this for multiple models.

**Comment by ncoop57** [comment]
No problem. Thank you for this awesome set of models and straightforward code! It has greatly helped my understanding of these models, especially VAEs.
